go/tools/cortex-jsonschema-exporter/alertmanager-config.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Opstrace, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/alecthomas/jsonschema"
+	"github.com/cortexproject/cortex/pkg/alertmanager"
+)
+
+//
+// Prints the json schema definition of the Cortex configuration to stdout.
+//
+func main() {
+	schema := jsonschema.Reflect(&alertmanager.Config{})
```

Comment:
@terrcin Can you tell me which struct you looked at when you created the schema in https://github.com/opstrace/opstrace/blob/main/packages/app/src/client/validation/alertmanager/config/types.ts ?

Reply:
@sreis I looked at the alertmanager config documentation itself and manually created those types based on that. Did it at the same time as the Yup validations.

Reply:
Ok. I'll try to find the corresponding golang struct.

Reply:
Changed to use the one that looked the most similar. 

@terrcin Any ideias how we can compare the autogenerated JSON schema with https://github.com/opstrace/opstrace/blob/main/packages/app/src/client/validation/alertmanager/config/schema.json ?

Reply:
@sreis maybe by eye at first. Do you have an example of the output somewhere I can look it? Also could do over zoom when we're next both online.